features/admin.applications.v1/components/forms/org-selective-share-with-all-roles.tsx

```diff
@@ -0,0 +1,562 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { TreeItem, TreeItemProps } from "@mui/x-tree-view";
```

Comment:
Due to time constraints we have used `x-tree-view` directly from mui-x. This will be imported from oxygen-ui in a follow-up PR